tests/testthat/test-labelled.R

```diff
@@ -1,7 +1,7 @@
 
-test_that("make_labelled applies labels to all elements of supertibble",{
+test_that("make_labelled applies labels to all elements of supertibble", {
   supertbl <- tibble::tribble(
-    ~ redcap_data, ~ redcap_metadata, ~ redcap_events,
+    ~redcap_data, ~redcap_metadata, ~redcap_events,
```

Comment:
How come here it gets rid of the space between variable and `~` but below it adds it? I guess because the application is different (variable vs function)?

Reply:
In the context of tribbles, there's no space after the tilde, in the context of anonymous functions there's a space.